src/utils/tagToSlug.js

```diff
@@ -0,0 +1,6 @@
+module.exports = tag =>
```

Comment:
feels like a wrong place for these utils ... in general feels like components, utils, etc should have structure now - blog, docs ... but again, not sure about best practices

Reply:
I agree in this case. Inlined this helper right into the component where it uses

Reply:
About utils structure we can think in the next PRs. Maybe we should separate them at least on usage sphere: gatsby node api/server/react side for example.

Reply:
It's too uses in `gatsby-node` (ie pages generation side) and in react component...